### ISSUE TITLE
Incorrect behavior of ManualJsonStringToJsonEncodeArrayRector

### ISSUE THREAD

**Issue by zingimmick** [rectorphp/rector#6637]
# Bug Report

| Subject        | Details              |
| :------------- | :--------------------|
| Rector version | last dev-master      |
| Installed as   | composer dependency  |

Rule `ManualJsonStringToJsonEncodeArrayRector` always convert manual JSON string to \Nette\Utils\Json::encode array  even if the class 'Nette\Utils\Json' not found.

## Minimal PHP Code Causing Issue

See https://getrector.org/demo/1ebfe3fd-7e9c-614c-9bb2-274ba6035927

```php
<?php

final class DemoFile
{
    public function run()
    {
        $someJsonAsString = '{"role_name":"admin","numberz":{"id":"10"}}';
    }
}
```

### Responsible rules

* `ManualJsonStringToJsonEncodeArrayRector`

## Expected Behavior

Any of the following options:

- Convert manual JSON string to json_encode array and let `JsonDecodeEncodeToNetteUtilsJsonDecodeEncodeRector` in [rector-nette](https://github.com/rectorphp/rector-nette) handle the rest.
- Convert manual JSON string to json_encode array if the class 'Nette\Utils\Json' not found.

**Comment by TomasVotruba** [comment]
Hi, thanks for reporting.

> Convert manual JSON string to json_encode array if the class 'Nette\Utils\Json' not found.

This would be ideal in this case IMO :+1: Could you propose a PR to change `JsonDecodeEncodeToNetteUtilsJsonDecodeEncodeRector`?

**Comment by zingimmick** [comment]
> Hi, thanks for reporting.
> 
> > Convert manual JSON string to json_encode array if the class 'Nette\Utils\Json' not found.
> 
> This would be ideal in this case IMO 👍 Could you propose a PR to change `ManualJsonStringToJsonEncodeArrayRector`?

I tried to fix this issue at rectorphp/rector-src#699, but how can I test it when the class `Nette\Utils\Json` exists?
